packages/evm/src/evm.ts

```diff
@@ -239,13 +239,13 @@ export class EVM {
    *
    * @param opts EVM engine constructor options
    */
-  static async create(opts: EVMOpts = {}): Promise<EVM> {
+  static async create(opts: EVMCreateOpts = {}): Promise<EVM> {
     const evm = new this(opts)
     await evm.init()
     return evm
   }
 
-  constructor(opts: EVMOpts) {
+  constructor(opts: EVMCreateOpts) {
```

Comment:
This option should for *not* be renamed. 😜

Needs to be (half-)reverted.

Reply:
I don't think I follow, why should this not be renamed? Should I introduce `EVMOpts = EVMCreateOpts`? :thinking:

Reply:
Because this is the other data structure for which we started this renaming in the first place???

So to recap: you named the thing in types.ts EVMOpts, which was the same name as the general options which we have here for the constructor. For that reason the name in types.ts should be named to something else. Am I missing something?

Reply:
Oh right, I completely missed this point. Note that the data structure itself has not changed (we do not import `EVMCreateOpts` from `types.ts`). I have addressed this in #2866

Reply:
Thanks, I was getting a bit desparte. 😂